mm2src/common/common.rs

```diff
@@ -747,6 +748,9 @@ pub fn writeln(line: &str) {
 
 pub fn small_rng() -> SmallRng { SmallRng::seed_from_u64(now_ms()) }
 
+#[inline(always)]
+pub fn os_rng(dest: &mut [u8]) { rand::rngs::OsRng.fill_bytes(dest); }
```

Comment:
The `fill_bytes` function can cause a panic if the getentropy syscall fails for any reason. Do you think this can ever happen?

Reply:
It can happen if the memory address is corrupt or the OS kernel deprecates/changes the syscall. Thanks for catching it.